src/utils/math.ts

```diff
@@ -122,31 +122,31 @@ export function forceFillColumnWidths(allColumns: any, expectedWidth: any, start
 
   for(let column of allColumns) {
     if(!column.canAutoResize) {
-      contentWidth += column.width;
+      contentWidth += +column.width;
     } else {
-      contentWidth += (column.$$oldWidth || column.width);
+      contentWidth += (+column.$$oldWidth || +column.width);
     }
   }
 
-  let remainingWidth = expectedWidth - contentWidth;
+  let remainingWidth = +expectedWidth - contentWidth;
   let additionWidthPerColumn = remainingWidth / columnsToResize.length;
   let exceedsWindow = contentWidth > expectedWidth;
 
   for(let column of columnsToResize) {
     if(exceedsWindow) {
-      column.width = column.$$oldWidth || column.width;
+      column.width = +column.$$oldWidth || +column.width;
     } else {
       if(!column.$$oldWidth) {
-        column.$$oldWidth = column.width;
+        column.$$oldWidth = +column.width;
       }
 
       const newSize = column.$$oldWidth + additionWidthPerColumn;
-      if(column.minWith && newSize < column.minWidth) {
-        column.width = column.minWidth;
-      } else if(column.maxWidth && newSize > column.maxWidth) {
-        column.width = column.maxWidth;
+      if(column.minWith && newSize < +column.minWidth) {
```

Comment:
Rather than do the coercion several times, lets do it once in the top of the function and just use those variables.

Or maybe even better and use a getter in the column object to do this automatically.